pkgs/by-name/ph/phraze/package.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "phraze";
+  version = "0.3.10";
+
+  src = fetchFromGitHub {
+    owner = "sts10";
+    repo = "phraze";
+    rev = "c88511d6f267076d8724c93d2067ac349957a0d0";
```

Comment:
Why not to use version here?

Reply:
@AndersonTorres  Fixed. Somewhere in the docs it mentions a full commit hash being "best", but the tag does make the most sense.